source/excerptexport/urls.py

```diff
@@ -1,23 +1,21 @@
 from django.conf.urls import patterns, url
 
-from excerptexport import views
-
 
 urlpatterns = patterns(
-    '',
-    url(r'^$', views.index, name='index'),
```

Comment:
reverse_lazy is your friend though, don't need the string references anymore, hence -> :-)